components/stratum/stratum_api.c

```diff
@@ -375,6 +378,9 @@ int STRATUM_V1_configure_version_rolling(int socket, int send_uid, uint32_t * ve
 
 static void debug_stratum_tx(const char * msg)
 {
+    // Record the timestamp when sending
+    gettimeofday(&last_tx_time, NULL);
```

Comment:
its more appropriate to use a monotonic clock for this instead of wall time. https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/system/esp_timer.html#_CPPv418esp_timer_get_timev